lua/cmp/config/default.lua

```diff
@@ -101,5 +101,7 @@ return function()
         winhighlight = 'FloatBorder:NormalFloat',
       },
     },
+
+    incseach_redraw_keys = '<Up><Down>'
```

Comment:
I don't think it's good to put here.

I think it should be hidden API. So `require('cmp.utils.misc').redraw.incsearch_redraw_keys = '<Up><Down>'` is better.

Reply:
Fixed.
I have `require('cmp.utils.misc').redraw.incsearch_redraw_keys = '<C-r><BS>'` and it works fine.